package.json

```diff
@@ -38,6 +38,9 @@
     "type": "git",
     "url": "https://github.com/Ashung/svg2vectordrawable.git"
   },
+  "scripts": {
+    "test": "npx jasmine spec/*.js"
```

Comment:
I couldn't find any existing tests, so only set up a minimal test setup to test out some code. So I'm definitely willing to change the tests to fit any other preferences.